driver/config/config.go

```diff
@@ -341,6 +343,7 @@ func New(ctx context.Context, l *logrusx.Logger, stdOutOrErr io.Writer, opts ...
 		configx.WithStderrValidationReporter(),
 		configx.OmitKeysFromTracing("dsn", "courier.smtp.connection_uri", "secrets.default", "secrets.cookie", "secrets.cipher", "client_secret"),
 		configx.WithImmutables("serve", "profiling", "log"),
+		configx.WithExceptImmutables("serve.public.cors.allowed_origins", "serve.admin.cors.allowed_origins"),
```

Comment:
Does admin even have cors?

Also what about cors enabled? I think that if cors is enabled and allowed origins is empty, it allows all origins. Not sure though.

Reply:
Yes correct, no origins means all origins.
To hot-reload "enabled", I guess we just always return false in the check function when it is not enabled?